about/index.md

```diff
@@ -13,91 +13,64 @@ description:
 ---
 
 <div class='flex md:flex-row flex-col-reverse rounded-md bg-indigo-500 p-3 md:h-80'>
-<p class='text-2xl px-4 my-auto text-white '>
+<p class='cover-description text-2xl px-4 my-auto text-white '>
 Hi, I am Geshan. I am a software engineer, with more than a decade of software engineering experience. Currently, I am living in Sydney, Australia serving THE ICONIC as a lead software engineer.
 </p>
 <img class="center rounded-full md:border-4 border-white h-44  md:h-5/6 m-auto " src="/images/geshan.jpg" title="Geshan Manandhar" alt="Geshan Manandhar">
 </div>
-<div class='mt-10 text-lg '>
-<p>
-I have a keen interest in REST architecture, microservices, and cloud computing. I am a language-agnostic software engineer who believes the value provided to the business is more important than the choice of language or framework.</p>
-<p class='mt-5'>
 
- <h2 class='text-2xl font-bold text-darkpurple'>Consciously evolving</h2>
+I have a keen interest in REST architecture, microservices, and cloud computing. I am a language-agnostic software engineer who believes the value provided to the business is more important than the choice of language or framework.
 
-I was born and lived the majority of my life in Kathmandu, Nepal. I have lived and worked in the Netherlands, U.A.E, and Australia. This multi-cultural experience has enriched my life in various aspects. It has also given me chances to work on software systems of varying scale that millions of customers have used, benefitted from, and can rely upon.
-
-</p>
-<p class='mt-5'>
+## Consciously evolving
```

Comment:
Changed it back to markdown format, except for the intro banner and feedback.

Reply:
About page as it is single you can do HTML on this one, not on posts :) as there are 100s.